dhall/src/Dhall/Parser/Expression.hs

```diff
@@ -718,6 +749,7 @@ env = do
         _ <- Text.Parser.Char.char '"'
         return a
 
+-- | Parse a local import without trailing whitespace
 localRaw :: Parser ImportType
```

Comment:
Maybe we should rename this to `localOnly` to be consistent with other parsers that ignore trailing whitespace. (Or rename those to `xyzRaw`)